src/includes/getting-started-config/dart.mdx

```diff
@@ -1,7 +1,7 @@
 ```dart
 import 'package:sentry/sentry.dart';
 
-Future<void> main() async {
+void main() async {
```

Comment:
I don't know about Dart, but in C# `async void` is an issue.

That's because now you've allowed to run async code and await for futures. But no future is retuned, so how can the caller await for it? Basically makes this a fire and forget operation.

In Flutter I belive a `void main() async` works and the app doesn't exit immediately which is an indication things are different, but then, how does it work?

When should you return Future and when should you not?

Reply:
every sample I found is `void main() async` for main only, so I think that's the trick, for all the other methods it should not be `void` but rather `Future` or `FutureOr<void>`

Reply:
@rxlabz do you know if it's a special case?

Reply:
Added just in case, thanks!